packages/react-components/src/link/src/useLink.ts

```diff
@@ -1,6 +1,32 @@
-import { cssModule, mergeClasses, normalizeSize, useAutoFocus, useMergedRefs } from "../../shared";
+import { ForwardedRef } from "react";
+import { MergedRef, Size, cssModule, mergeClasses, normalizeSize, useAutoFocus, useMergedRefs } from "../../shared";
 import { isNumber } from "lodash";
 
+export interface UseLinkProps {
```

Comment:
Should this be the LinkProps?

Reply:
Not exactly, It will of course be a subset of the props but it could differ at some point